### ISSUE TITLE
Installation fails without a terminal

### ISSUE THREAD

**Issue by dannysauer** [awslabs/git-secrets#220]
https://github.com/awslabs/git-secrets/blob/80230afa8c8bdeac766a0fece36f95ffaa0be778/git-secrets#L208

There's no particularly good reason to depend on `say` _and_ to require a valid terminal just to print out a green checkmark symbol upon successful installation. :)

```
Run git secrets --install
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
/usr/local/bin/git-secrets: line 208: say: command not found
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
/usr/local/bin/git-secrets: line 208: say: command not found
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
/usr/local/bin/git-secrets: line 208: say: command not found
Error: Process completed with exit code 127.
```

**Comment by cahrens** [comment]
As of about 5 days ago, I am suddenly hitting the problem of `say` not being installed (though oddly in our project it happens sporadically). Has anyone figured out a workaround to this problem?

**Comment by dannysauer** [comment]
A couple of workarounds:
You could `ln -s "$(which echo)" say` in your $PATH and this should continue working. :)
Or upgrade to my branch / apply the diff in the PR mentioned before running the install.

**Comment by cahrens** [comment]
@dannysauer unfortunately `ln -s "$(which echo)"` did not work. Thank you for the suggestion though!

**Comment by cahrens** [comment]
This is possibly related: https://github.com/git/git/commit/5b893f7d81eb7feb43662ed8663e2af76a76b4c8

**Comment by dannysauer** [comment]
@cahrens the idea was to put say in your path, something like /usr/local/bin/say. :)

`sudo ln -s "$(which echo)" /usr/local/bin/say`

**Comment by Greg-Myers-SB** [comment]
Oh this explains why my Mac started reading the git-secrets out loud! 🤣 
Since git no longer has a `say` command, when git-secrets is installing on MacOS, it now reads the secrets hook out via the voice over utility. Looks like I'm using git <= 2.37 until this is changed.

**Comment by kdenhartog** [comment]
for those who are on MacOS and still encountering this problem (where you get the voice over utility issue), you'll need to use the `make install` method of installation. The brew cask has not been updated yet.

**Comment by Greg-Myers-SB** [comment]
@kees-closed This issue was fixed in #221 
You may be encountering a new bug or may not have the latest version available in your OS' repositories.

**Comment by alvincrespo** [comment]
When I tried to do `which echo` on my Mac, I got the following:

```sh
which echo
echo: shell built-in command
```

To resolve this, I did `type -a echo`, which gave me:

```sh
echo is a shell builtin
echo is /bin/echo
```

I then symlinked it:

```sh
sudo ln -s "/bin/echo" /usr/local/bin/say
```

Now, when I run `git secrets` I no longer get yelled at:

```sh
git secrets --install -f
✓ Installed commit-msg hook to .git/hooks/commit-msg
✓ Installed pre-commit hook to .git/hooks/pre-commit
✓ Installed prepare-commit-msg hook to .git/hooks/prepare-commit-msg
```
